ui/src/app/components/runtime-configuration-panel.tsx

```diff
@@ -586,3 +579,36 @@ export const RuntimeConfigurationPanel = fp.flow(
     );
   }
 );
+
+export interface RuntimeConfigurationPanelProps {
```

Comment:
please move this back above RuntimeConfigurationPanelContent, because it pertains to both